kimminwoo/likelion-assignment-crud/src/main/java/com/kimminwoo/likelionassignmentcrud/student/api/StudentController.java

```diff
@@ -0,0 +1,37 @@
+package com.kimminwoo.likelionassignmentcrud.student.api;
+
+
+import com.kimminwoo.likelionassignmentcrud.student.api.dto.request.StudentSaveRequestDto;
+import com.kimminwoo.likelionassignmentcrud.student.api.dto.response.StudentInfoResponseDto;
+import com.kimminwoo.likelionassignmentcrud.student.api.dto.response.StudentListResponseDto;
+import com.kimminwoo.likelionassignmentcrud.student.application.StudentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/student")
+public class StudentController {
+
+    private final StudentService studentService;
+
+    @PostMapping("/save")
+    public ResponseEntity<String> studentSave(@RequestBody StudentSaveRequestDto studentSaveRequestDto) {
+        studentService.studentSave(studentSaveRequestDto);
+        return new ResponseEntity<>("사용자 저장!", HttpStatus.CREATED);
+    }
+
+    @GetMapping("/all")
+    public ResponseEntity<StudentListResponseDto> studentFindAll() {
+        StudentListResponseDto studentListResponseDto = studentService.studentFindAll();
+        return new ResponseEntity<>(studentListResponseDto, HttpStatus.OK);
```

Comment:
```suggestion
        return new ResponseEntity<>(studentService.studentFindAll(), HttpStatus.OK);
```
굳이 studentListResponseDto 변수를 따로 생성하지 않고, 위와 같이 바로 반환하면 코드가 더 간결하고 불필요한 변수 생성을 줄일 수 있어 가독성이 좋아집니다! 아래 studentFindOne 메소드에서도 마찬가지겠죠?
메모리 측면에서도 조금 더 효율적인 방식이니 참고해주세요 😊